### ISSUE TITLE
Unusual output for the address list -k command

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#194]
I was about to test the `address cache` command with the `-k` flag so I've tried to list the keyring addresses I have using the `address list -k` command, but got the following output:
```
 Error: Label donor_address1 was not stored in the keyring.
 Error: Label new_after_refill9 was not stored in the keyring.
 Error: Label origin_address was not stored in the keyring.
 Error: Label new_keyring was not stored in the keyring.
 Error: Label keyring_address was not stored in the keyring.
 Error: Label new_one was not stored in the keyring.
 Error: Label donor_address8 was not stored in the keyring.
 Error: Label donor_address1.2.1 was not stored in the keyring.
/usr/lib/python3.12/site-packages/jeepney/low_level.py:185: ResourceWarning: unclosed <socket.socket fd=8, family=1, type=1, proto=0, raddr=/run/user/1000/bus>
  if any(isinstance(f, DictEntry) for f in fields):
ResourceWarning: Enable tracemalloc to get the object allocation traceback
 Error: Label donor_address1.2 was not stored in the keyring.
 Error: Label donor_address was not stored in the keyring.
 Error: Label origin_address2 was not stored in the keyring.
 Error: Label main was not stored in the keyring.
s+Balances of addresses with labels in wallet:-------------+---------------+------------+-------------+
| Label              | Address                            | tslm          | PoB tokens | dPoD tokens |
+--------------------+------------------------------------+---------------+------------+-------------+
| donor_address1     | None                               | 620003.192122 | 0          | 0           |
| new_after_refill9  | None                               | 620003.192122 | 0          | 0           |
| origin_address     | None                               | 620003.192122 | 0          | 0           |
| new_keyring        | None                               | 620003.192122 | 0          | 0           |
| keyring_address    | None                               | 620003.192122 | 0          | 0           |
| new_one            | None                               | 620003.192122 | 0          | 0           |
| donor_address8     | None                               | 620003.192122 | 0          | 0           |
| donor_address1.2.1 | None                               | 620003.192122 | 0          | 0           |
| donor_address1.2   | None                               | 620003.192122 | 0          | 0           |
| donor_address      | None                               | 620003.192122 | 0          | 0           |
| origin_address2    | None                               | 620003.192122 | 0          | 0           |
| main               | None                               | 620003.192122 | 0          | 0           |
|                    | mpKx1QBBFSPY3b7CjrR4Mvcq4t1tHiAEcz | 39.75         | 20.0       | 0           |
|                    | mgpiP2Dc5QweKFS55HvRqsWyQ6PMXJCVCk | 376.02        | 0          | 0           |
|                    | mfq11jH49F2jqUMrVpGZhtyeDC88Vk9Vs8 | 4883.56       | 2400.0     | 0           |
|                    | mkxLRzTvrEpPbFxkp4tnRGXQ6UKcKH25TA | 198.54        | 110.0      | 0           |
|                    | msAzW8dLyCNhnashwThAo2rUNKBycqt7yv | 12.952825     | 0          | 0           |
|                    | mwZdXxq6k2xF8ucj2FqeouMJpu9CPdpg68 | 0.01          | 0          | 0           |
|                    | moLzhbnv9KKMrF991g31LeuDwddEWmCuHv | 0.01          | 0          | 0           |
|                    | n2MfjNpVG2mWAHwKM7ijGCwmD91eXUTT7j | 0.01          | 0          | 0           |
|                    | mgUkRF2eosnFQpMKqAYxuzVLxccTPyNxYw | 0.03          | 111.111    | 0           |
|                    | mtYvCVBtEayA5y6szGSrSgGwHQfe44Bgh2 | 0.03          | 0          | 0           |
|                    | mx5MdsenFDZufFuwT9ND7BKQBzS4Hy9YUP | 95808.951397  | 73410.0    | 0           |
|                    | mq1jZAiu8NoL3U3Ed7YiDdfjmY5t6Zphvq | 99.87         | 9999.0     | 0           |
|                    | mvh35g4RwZqEPcxxctp5qtJcngx5hWXhJj | 0.01          | 0          | 0           |
|                    | mxuqE6UWREDuWuMkBWoz1F8jfZur5CM3ZH | 3908.83       | 1700.0     | 0           |
|                    | miHhMLaMWubq4Wx6SdTEqZcUHEGp8RKMZt | 0.2           | 0          | 0           |
|                    | mokMMyxxMD15dWbfBWtfQNQ8cSfqVas4je | 0.01          | 0          | 0           |
|                    | n2nSbZYLV77NFHSD3P4ZdFKyp65rtTHP1b | 4103.19       | 102366.0   | 0           |
|                    | mskxtnWBJ1bBYZxmHe8rTR5H8ayZEXvG35 | 426472.51     | 142690.0   | 0           |
|                    | n3pFtPbzAfMxpK8X25MLSExNB3r5Xe3MhE | 0.02          | 0          | 0           |
|                    | mhQqLnhv856S44xAsmG8rNBWtj4dqvYD8X | 0.1           | 0          | 0           |
|                    | mfemhfkdD9tZG7Lukv2dT7vdq8d5xYFQCX | 5872.710554   | 1900.0     | 0           |
|                    | mwgpxrjnzXxfXSdZaD6WvtH1122B1fwMv4 | 23126.97      | 1800.0     | 0           |
|                    | n3qWzoBX24abCSq8hMMzmCb8dh5ePnhdmL | 451.3         | 1.0        | 0           |
|                    | mrnZpB5Wm1FDBBcZ1s1pdPHpQgzaLRteSr | 4499.71       | 2000.0     | 0           |
|                    | mkwJijAwqXNFcEBxuC93j3Kni43wzXVuik | 0.06          | 0          | 0           |
|                    | mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2 | 0.03          | 0.7        | 0           |
|                    | n3e58QHqUD5gvudHtXzdw4A9otuB3cZvW6 | 0.01          | 0          | 0           |
|                    | mz6HENDiSFPYqsi7wDZ33p5jt7j9LFo6qr | 0.02          | 0          | 0           |
|                    | n3MtPoPaAREU5GEhAErBPuju83bVog2opz | 3720.294      | 4175.7977  | 0           |
|                    | mnYTcGF8vVuSAYeoxxNaFWX94QUFM92YZf | 730.92        | 30.0       | 0           |
|                    | mjXqewNUxUCW4v6bQ7CzhRAVMLHmQ6ktxN | 3849.73       | 2100.0     | 0           |
|                    | mwvsHNwNQ8eiRFGHNqnpD7EdoFk5zzAdYH | 1153.23       | 100.0      | 0           |
|                    | myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc | 0.03          | 0.6        | 0           |
|                    | mixcqdAy4aFtuBV7uRebqBTjiFWCcS4UEp | 30103.55      | 61598.1    | 0           |
|                    | n4di1EHkFnSGYpJCLNpHfheAPhuTm8VG2j | 0.06          | 0          | 0           |
|                    | mjTBaL9aBxyaqvPz6Hha9YowdQEZhZDL73 | 0.01          | 0          | 0           |
|                    | mijeUJNpC1Kqidne8aq7KbXDb5fApSWd39 | 60.36         | 1.0        | 0           |
+--------------------+------------------------------------+---------------+------------+-------------+
```
So I've run `address set keyring_new mq1jZAiu8NoL3U3Ed7YiDdfjmY5t6Zphvq -k` and then ` address list -k` and the `address list -k | grep keyring_new` command outputs as following:
```
| keyring_new        | None                               | 620054.152122 | 0          | 0           |
```
Moreover after I've run `address list -k` the `address list` command has begun outputting the following:
```
+Balances of addresses with labels in wallet:--+--------------+------------+-------------+
| Label   | Address                            | tslm         | PoB tokens | dPoD tokens |
+---------+------------------------------------+--------------+------------+-------------+
| another | mwLKewF56R8Rje2tAuQRJzDxS9XC1a1q23 | 0            | 0.4        | 0           |
| another | mijeUJNpC1Kqidne8aq7KbXDb5fApSWd39 | 60.36        | 1.0        | 0           |
| donor   | mukgdQJzBzCkMVjmUNgcmDJD5cwY3zzua6 | 0            | 1.0        | 0           |
| donor   | n4AoPomtAQRxsTc2TDMFMsP2PonhZ2vVAz | 0            | 4.0        | 0           |
| donor   | mx5MdsenFDZufFuwT9ND7BKQBzS4Hy9YUP | 95808.951397 | 73410.0    | 0           |
| donor   | mhg25VziknSgQLBg6uCMBXsT8VpwbaQUJR | 0            | 10.0       | 0           |
| donor   | mzHSKpCYK2J4Np6cVimPQsJufU7ZwesrcL | 0            | 0          | 0           |
| donor   | mskwPWmHNAV2yfKPDCs4Yt6WvSpfGbA64e | 0            | 0          | 0           |
| donor   | mwgpxrjnzXxfXSdZaD6WvtH1122B1fwMv4 | 23126.97     | 1800.0     | 0           |
| donor   | mfemhfkdD9tZG7Lukv2dT7vdq8d5xYFQCX | 5872.710554  | 1900.0     | 0           |
| donor   | mpKx1QBBFSPY3b7CjrR4Mvcq4t1tHiAEcz | 39.75        | 20.0       | 0           |
| donor   | mrnZpB5Wm1FDBBcZ1s1pdPHpQgzaLRteSr | 4499.71      | 2000.0     | 0           |
| donor   | mq1jZAiu8NoL3U3Ed7YiDdfjmY5t6Zphvq | 99.87        | 9999.0     | 0           |
| donor   | n1xC4Dkp48mbsJsMV3yYEho6adBmbjgqNV | 0            | 10.0       | 0           |
| donor   | mskxtnWBJ1bBYZxmHe8rTR5H8ayZEXvG35 | 426472.51    | 142690.0   | 0           |
| donor   | mg59ZLqztR5zWQB9HknPNHFH4PHwcq4Ee2 | 0            | 0          | 0           |
| donor   | mfq11jH49F2jqUMrVpGZhtyeDC88Vk9Vs8 | 4883.56      | 2400.0     | 0           |
| flag    | mgUkRF2eosnFQpMKqAYxuzVLxccTPyNxYw | 0.03         | 111.111    | 0           |
| fresh   | n4UW9h3QvBRxdF9prmT8cP8p13Vm8VYJn9 | 0            | 0.2        | 0           |
| fresh   | mue63g5tHoM4qLQNBYzF7kSHbj8yvKfVYz | 0            | 0          | 0           |
| fresh   | mr99o4xUWMaVNmueFuqHMvZwRPxkLiQe8U | 0            | 0          | 0           |
| fresh   | mpo8FqEPH5iSakV9GZAhRwTrvHeJ98CWBv | 0            | 0          | 0           |
| fresh   | mwUJwJePvN3cNPaNB3LJkTLWenAXQV9ekw | 0            | 0          | 0           |
| fresh   | n4CLjrpKA5oLuEn63M3iWfGP7fyQvxkg6b | 0            | 0          | 0           |
| fresh   | myGqHjaG4BVnh51rh1g6w8mH2oJQZ3QCJP | 0            | 0          | 0           |
| fresh   | mwahMEM12h4m3nuq58MwetYDYPbYUr918i | 0            | 0          | 0           |
| fresh   | mtJoREAk9dRQQXR6KNAkXb1gYAd5LCkML9 | 0            | 1.0        | 0           |
| fresh   | mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2 | 0.03         | 0.7        | 0           |
| keyring | mw9gjLtQVhu34YKep7vrZJDiong69Zmyid | 0            | 0          | 0           |
| locator | mmWqnW9DkPq2Dosm8ikmNvd8xfaeynp6iw | 2704.648007  | 0          | 0           |
| locator | myu48LbhEumui6yMkdD5DgYYf7STgEtF7Y | 2420.62      | 0          | 0           |
| locator | mgpKAUYnTyo9FKEhHKYeXMzBFkLdPBuW8j | 556.55       | 0          | 0           |
| main    | mw6J8BWwsbERhCmVYPdS4Kk5W21FuK3Qt3 | 0            | 1.0        | 0           |
| new     | mwvsHNwNQ8eiRFGHNqnpD7EdoFk5zzAdYH | 1153.23      | 100.0      | 0           |
| new     | mnYTcGF8vVuSAYeoxxNaFWX94QUFM92YZf | 730.92       | 30.0       | 0           |
| new     | mk9WFSTqgtTJbwb5g6pGpByTRbYKiSaX3w | 0            | 3.0        | 0           |
| new     | n3qWzoBX24abCSq8hMMzmCb8dh5ePnhdmL | 451.3        | 1.0        | 0           |
| new     | n454R3cUohDkjPvfkpjMZysWjp6FkR8mfC | 598.85       | 0          | 0           |
| new     | n1zrFcgeDhcMP3PGyttiKviAW92TMrQf6r | 632.96       | 0          | 0           |
| new     | mffwdnxCEFgM2GoZD1Cd6yibPB43RewaVA | 604.86       | 0          | 0           |
| new     | n3eHVFDPbBHuq9CrNjirJ33GNTQuMnRZVy | 367.21       | 0          | 0           |
| new     | mkxLRzTvrEpPbFxkp4tnRGXQ6UKcKH25TA | 198.54       | 110.0      | 0           |
| new     | mtYvCVBtEayA5y6szGSrSgGwHQfe44Bgh2 | 0.03         | 0          | 0           |
| new     | mtmKbmMHdcXXepscuNrwAfjsKLeZzdZdya | 2639.63      | 1203.0     | 7.0         |
| no      | mqZ5nqs6bpaAqEoEV9jVFfwsM3hBZD4C3J | 0            | 0          | 0           |
| nw      | mkbU3SGSMxt4tjVepPHGywWWJ7gYGrhoBG | 0            | 0          | 0           |
| old     | n2nSbZYLV77NFHSD3P4ZdFKyp65rtTHP1b | 4103.19      | 102366.0   | 0           |
| origin  | mxuqE6UWREDuWuMkBWoz1F8jfZur5CM3ZH | 3908.83      | 1700.0     | 0           |
| origin  | mfkwJpC1RaEAPrh6bNk849jFyzMgBA3td1 | 0            | 0          | 0           |
| origin  | n3MtPoPaAREU5GEhAErBPuju83bVog2opz | 3720.294     | 4175.7977  | 0           |
| origin  | mjXqewNUxUCW4v6bQ7CzhRAVMLHmQ6ktxN | 3849.73      | 2100.0     | 0           |
| origin  | mrnrAyYBG9gNzdAiZ6aRDC3tBkrCsjTnSi | 0            | 100.0      | 0           |
| second  | mjAykGXoJu21VZLr5JymuQXFCXyzhgn3Cb | 0            | 1.0        | 0           |
| super   | msAzW8dLyCNhnashwThAo2rUNKBycqt7yv | 12.952825    | 0          | 0           |
| test    | mgpiP2Dc5QweKFS55HvRqsWyQ6PMXJCVCk | 376.02       | 0          | 0           |
| usage   | myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc | 0.03         | 0.6        | 0           |
|         | n3pFtPbzAfMxpK8X25MLSExNB3r5Xe3MhE | 0.02         | 0          | 0           |
|         | moLzhbnv9KKMrF991g31LeuDwddEWmCuHv | 0.01         | 0          | 0           |
|         | mhQqLnhv856S44xAsmG8rNBWtj4dqvYD8X | 0.1          | 0          | 0           |
|         | miHhMLaMWubq4Wx6SdTEqZcUHEGp8RKMZt | 0.2          | 0          | 0           |
|         | n4di1EHkFnSGYpJCLNpHfheAPhuTm8VG2j | 0.06         | 0          | 0           |
|         | n2MfjNpVG2mWAHwKM7ijGCwmD91eXUTT7j | 0.01         | 0          | 0           |
|         | mixcqdAy4aFtuBV7uRebqBTjiFWCcS4UEp | 30103.55     | 61598.1    | 0           |
|         | mokMMyxxMD15dWbfBWtfQNQ8cSfqVas4je | 0.01         | 0          | 0           |
|         | mz6HENDiSFPYqsi7wDZ33p5jt7j9LFo6qr | 0.02         | 0          | 0           |
|         | mjTBaL9aBxyaqvPz6Hha9YowdQEZhZDL73 | 0.01         | 0          | 0           |
|         | mwZdXxq6k2xF8ucj2FqeouMJpu9CPdpg68 | 0.01         | 0          | 0           |
|         | n3e58QHqUD5gvudHtXzdw4A9otuB3cZvW6 | 0.01         | 0          | 0           |
|         | mkwJijAwqXNFcEBxuC93j3Kni43wzXVuik | 0.06         | 0          | 0           |
|         | mvh35g4RwZqEPcxxctp5qtJcngx5hWXhJj | 0.01         | 0          | 0           |
+---------+------------------------------------+--------------+------------+-------------+
```
While previously it was like this:
```
+Balances of addresses with labels in wallet:---------------+--------------+------------+-------------+
| Label                | Address                            | tslm         | PoB tokens | dPoD tokens |
+----------------------+------------------------------------+--------------+------------+-------------+
| another_fresh        | mwLKewF56R8Rje2tAuQRJzDxS9XC1a1q23 | 0.0          | 0.4        | 0           |
| another_test_address | mijeUJNpC1Kqidne8aq7KbXDb5fApSWd39 | 60.36        | 1.0        | 0           |
| donor_address        | mukgdQJzBzCkMVjmUNgcmDJD5cwY3zzua6 | 0.0          | 1.0        | 0           |
| donor_address0.2     | n4AoPomtAQRxsTc2TDMFMsP2PonhZ2vVAz | 0.0          | 4.0        | 0           |
| donor_address1       | mx5MdsenFDZufFuwT9ND7BKQBzS4Hy9YUP | 94967.831397 | 73410.0    | 0           |
| donor_address1.1     | mhg25VziknSgQLBg6uCMBXsT8VpwbaQUJR | 0.0          | 10.0       | 0           |
| donor_address1.2     | mzHSKpCYK2J4Np6cVimPQsJufU7ZwesrcL | 0.0          | 0          | 0           |
| donor_address1.2.1   | mskwPWmHNAV2yfKPDCs4Yt6WvSpfGbA64e | 0.0          | 0          | 0           |
| donor_address2       | mwgpxrjnzXxfXSdZaD6WvtH1122B1fwMv4 | 22879.81     | 1800.0     | 0           |
| donor_address3       | mfemhfkdD9tZG7Lukv2dT7vdq8d5xYFQCX | 5817.970554  | 1900.0     | 0           |
| donor_address3.2     | mpKx1QBBFSPY3b7CjrR4Mvcq4t1tHiAEcz | 39.75        | 20.0       | 0           |
| donor_address4       | mrnZpB5Wm1FDBBcZ1s1pdPHpQgzaLRteSr | 4372.24      | 2000.0     | 0           |
| donor_address5       | mq1jZAiu8NoL3U3Ed7YiDdfjmY5t6Zphvq | 99.87        | 9999.0     | 0           |
| donor_address6       | n1xC4Dkp48mbsJsMV3yYEho6adBmbjgqNV | 0.0          | 10.0       | 0           |
| donor_address7       | mskxtnWBJ1bBYZxmHe8rTR5H8ayZEXvG35 | 417881.17    | 142690.0   | 0           |
| donor_address8       | mg59ZLqztR5zWQB9HknPNHFH4PHwcq4Ee2 | 0.0          | 0          | 0           |
| donor_address9       | mfq11jH49F2jqUMrVpGZhtyeDC88Vk9Vs8 | 4773.46      | 2400.0     | 0           |
| flag_l               | mgUkRF2eosnFQpMKqAYxuzVLxccTPyNxYw | 0.03         | 111.111    | 0           |
| fresh_after_refill   | n4UW9h3QvBRxdF9prmT8cP8p13Vm8VYJn9 | 0.0          | 0.2        | 0           |
| fresh_after_refill1  | mue63g5tHoM4qLQNBYzF7kSHbj8yvKfVYz | 0.0          | 0          | 0           |
| fresh_after_refill2  | mr99o4xUWMaVNmueFuqHMvZwRPxkLiQe8U | 0.0          | 0          | 0           |
| fresh_after_refill3  | mpo8FqEPH5iSakV9GZAhRwTrvHeJ98CWBv | 0.0          | 0          | 0           |
| fresh_after_refill4  | mwUJwJePvN3cNPaNB3LJkTLWenAXQV9ekw | 0.0          | 0          | 0           |
| fresh_after_refill5  | n4CLjrpKA5oLuEn63M3iWfGP7fyQvxkg6b | 0.0          | 0          | 0           |
| fresh_after_refill6  | myGqHjaG4BVnh51rh1g6w8mH2oJQZ3QCJP | 0.0          | 0          | 0           |
| fresh_after_refill7  | mwahMEM12h4m3nuq58MwetYDYPbYUr918i | 0.0          | 0          | 0           |
| fresh_after_refill8  | mtJoREAk9dRQQXR6KNAkXb1gYAd5LCkML9 | 0.0          | 1.0        | 0           |
| fresh_after_refill9  | mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2 | 0.03         | 0.7        | 0           |
| keyring_address      | mw9gjLtQVhu34YKep7vrZJDiong69Zmyid | 0.0          | 0          | 0           |
| locator_test         | mmWqnW9DkPq2Dosm8ikmNvd8xfaeynp6iw | 2704.648007  | 0          | 0           |
| locator_test1        | myu48LbhEumui6yMkdD5DgYYf7STgEtF7Y | 2420.62      | 0          | 0           |
| locator_test2        | mgpKAUYnTyo9FKEhHKYeXMzBFkLdPBuW8j | 556.55       | 0          | 0           |
| main                 | mw6J8BWwsbERhCmVYPdS4Kk5W21FuK3Qt3 | 0.0          | 1.0        | 0           |
| new_after_refill     | mwvsHNwNQ8eiRFGHNqnpD7EdoFk5zzAdYH | 1153.23      | 100.0      | 0           |
| new_after_refill1    | mnYTcGF8vVuSAYeoxxNaFWX94QUFM92YZf | 730.92       | 30.0       | 0           |
| new_after_refill2    | mk9WFSTqgtTJbwb5g6pGpByTRbYKiSaX3w | 0.0          | 3.0        | 0           |
| new_after_refill3    | n3qWzoBX24abCSq8hMMzmCb8dh5ePnhdmL | 451.3        | 1.0        | 0           |
| new_after_refill4    | n454R3cUohDkjPvfkpjMZysWjp6FkR8mfC | 598.85       | 0          | 0           |
| new_after_refill5    | n1zrFcgeDhcMP3PGyttiKviAW92TMrQf6r | 632.96       | 0          | 0           |
| new_after_refill6    | mffwdnxCEFgM2GoZD1Cd6yibPB43RewaVA | 604.86       | 0          | 0           |
| new_after_refill7    | n3eHVFDPbBHuq9CrNjirJ33GNTQuMnRZVy | 367.21       | 0          | 0           |
| new_after_refill8    | mkxLRzTvrEpPbFxkp4tnRGXQ6UKcKH25TA | 198.54       | 110.0      | 0           |
| new_after_refill9    | mtYvCVBtEayA5y6szGSrSgGwHQfe44Bgh2 | 0.03         | 0          | 0           |
| new_one              | mtmKbmMHdcXXepscuNrwAfjsKLeZzdZdya | 2639.63      | 1203.0     | 7.0         |
| no_transactions?     | mqZ5nqs6bpaAqEoEV9jVFfwsM3hBZD4C3J | 0.0          | 0          | 0           |
| nw_1                 | mkbU3SGSMxt4tjVepPHGywWWJ7gYGrhoBG | 0.0          | 0          | 0           |
| old_default          | n2nSbZYLV77NFHSD3P4ZdFKyp65rtTHP1b | 3994.83      | 102366.0   | 0           |
| origin_address1      | mxuqE6UWREDuWuMkBWoz1F8jfZur5CM3ZH | 3908.83      | 1700.0     | 0           |
| origin_address2      | mfkwJpC1RaEAPrh6bNk849jFyzMgBA3td1 | 0.0          | 0          | 0           |
| origin_address4      | n3MtPoPaAREU5GEhAErBPuju83bVog2opz | 3618.504     | 4175.7977  | 0           |
| origin_address5      | mjXqewNUxUCW4v6bQ7CzhRAVMLHmQ6ktxN | 3794.58      | 2100.0     | 0           |
| origin_address7      | mrnrAyYBG9gNzdAiZ6aRDC3tBkrCsjTnSi | 0.0          | 100.0      | 0           |
| second_one           | mjAykGXoJu21VZLr5JymuQXFCXyzhgn3Cb | 0.0          | 1.0        | 0           |
| super_new            | msAzW8dLyCNhnashwThAo2rUNKBycqt7yv | 12.952825    | 0          | 0           |
| test_label           | mgpiP2Dc5QweKFS55HvRqsWyQ6PMXJCVCk | 376.02       | 0          | 0           |
| usage_checked        | myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc | 0.03         | 0.6        | 0           |
|                      | mjTBaL9aBxyaqvPz6Hha9YowdQEZhZDL73 | 0.01         | 0          | 0           |
|                      | mz6HENDiSFPYqsi7wDZ33p5jt7j9LFo6qr | 0.02         | 0          | 0           |
|                      | n4di1EHkFnSGYpJCLNpHfheAPhuTm8VG2j | 0.06         | 0          | 0           |
|                      | mwZdXxq6k2xF8ucj2FqeouMJpu9CPdpg68 | 0.01         | 0          | 0           |
|                      | n2MfjNpVG2mWAHwKM7ijGCwmD91eXUTT7j | 0.01         | 0          | 0           |
|                      | mixcqdAy4aFtuBV7uRebqBTjiFWCcS4UEp | 29816.62     | 61598.1    | 0           |
|                      | n3pFtPbzAfMxpK8X25MLSExNB3r5Xe3MhE | 0.02         | 0          | 0           |
|                      | miHhMLaMWubq4Wx6SdTEqZcUHEGp8RKMZt | 0.2          | 0          | 0           |
|                      | n3e58QHqUD5gvudHtXzdw4A9otuB3cZvW6 | 0.01         | 0          | 0           |
|                      | mhQqLnhv856S44xAsmG8rNBWtj4dqvYD8X | 0.1          | 0          | 0           |
|                      | mvh35g4RwZqEPcxxctp5qtJcngx5hWXhJj | 0.01         | 0          | 0           |
|                      | mkwJijAwqXNFcEBxuC93j3Kni43wzXVuik | 0.06         | 0          | 0           |
|                      | mokMMyxxMD15dWbfBWtfQNQ8cSfqVas4je | 0.01         | 0          | 0           |
|                      | moLzhbnv9KKMrF991g31LeuDwddEWmCuHv | 0.01         | 0          | 0           |
+----------------------+------------------------------------+--------------+------------+-------------+
```

**Comment by d5000** [comment]
In the latest two commits I have fixed the first issue alongside other improvements of the `address list` command.

I couldn't reproduce the second issue - if I understand correctly, after one run of `address list -k` you got also a different output with `address list` without `-k`?

It's possible that you just had updated to a buggy intermediate version when I was still working on the command, so can you re-check with the latest code if this still happens? It's difficult for me to imagine the logic behind that issue, because the keyring labels and the entries from the extended configuration file normally cannot affect each other.

**Comment by buhtignew** [comment]
> In the latest two commits I have fixed the first issue alongside other improvements of the address list command.

The issue is solved indeed.

> I couldn't reproduce the second issue - if I understand correctly, after one run of address list -k you got also a different output with address list without -k?

Correct

> It's possible that you just had updated to a buggy intermediate version when I was still working on the command

I think your guess is true.

> ...can you re-check with the latest code if this still happens? 

The output of `address list` hasn't changed, it still cuts the all the words of the label except the first one, although the labels aren't changed: if I want to use them I should utilize the full version, not the `address list`'s.
I remember we've got some similar issue in the beginning of our testings. Maybe I should do some kind of scanning in order to restore the previous appearance of the addresses?

**Comment by d5000** [comment]
I found the bug and fixed it (commit 59d6bd0). It had to do with the ways the labels are "cut" because they also include the network in the configuration file, and I had added a new function as part of the changes in the last week, which had a flaw.

As the bug was obvious, testing is not needed. If there are no further questions this can be closed.
